torch/_dynamo/eval_frame.py

```diff
@@ -289,6 +397,7 @@ def __init__(
         export=False,
         dynamic=None,
         compiler_config=None,
+        save_config=True,
```

Comment:
Can you explain somewhere what this kwarg does?

Reply:
Explained here: https://github.com/pytorch/pytorch/pull/111074#issuecomment-1759512216

Reply:
Oh if you mean in the code comments then sure.